Neos.Neos/Classes/AssetUsage/Dto/AssetIdAndOriginalAssetId.php

```diff
@@ -9,6 +9,7 @@
 /**
  * @internal
  */
+
```

Comment:
```suggestion
```

imo correct white space

Reply:
But htis wasn't even changed in this PR?

Reply:
Wait it was, why was this whitespace added in the first place, not by me IMHO.